src/comparers/compareNFTs.ts

```diff
@@ -11,7 +11,10 @@ import {
   NftDiff,
   NftFileData,
```

Comment:
Bit of a nitpick, but there is inconsistency with the use of NFTs, Nfts, and nfts.